test/hook/context/generator.go

```diff
@@ -34,14 +35,15 @@ func convertBindingContexts(bindingContexts []BindingContext) (string, error) {
 }
 
 type BindingContextController struct {
-	HookCtrl     controller.HookController
-	HookMap      map[string]string
-	HookConfig   string
-	InitialState string
-	Controller   StateController
-	Manager      manager.KubeEventsManager
-	Context      context.Context
-	Cancel       context.CancelFunc
+	HookCtrl        controller.HookController
+	HookMap         map[string]string
+	HookConfig      string
+	InitialState    string
+	Controller      StateController
+	BindingManager  bindingmanager.KubeEventsManager
```

Comment:
Why bingingmanager? `schedule` is also a bingind ;)

https://github.com/flant/shell-operator/blob/master/pkg/hook/hook_manager.go#L41-L42